hack/common.sh

```diff
@@ -2,7 +2,9 @@
 
 # Location of the manifests file
 MANIFEST_LOC=bundle/
-WMCO_DEPLOY_NAMESPACE=openshift-windows-machine-config-operator
+
+# define namespace
+declare -r WMCO_DEPLOY_NAMESPACE=openshift-windows-machine-config-operator
```

Comment:
https://tldp.org/LDP/abs/html/declareref.html

For anyone else who hadn't seen this before

Reply:
I'm thinking we should also use this definition here:
https://github.com/openshift/windows-machine-config-operator/blob/master/hack/run-ci-e2e-test.sh#L16
https://github.com/openshift/windows-machine-config-operator/blob/master/hack/run-ci-e2e-test.sh#L32

This e2e `source`'s hack/common.sh so the variable should be available